config/sample_config.txt

```diff
@@ -246,6 +246,11 @@ compare:
     # draw secondary structure on contact map plots
     draw_secondary_structure: True
 
+    # search using hmmbuild/hmmsearch
+    # be aware that if you use this option then
+    # you may need to set the thresholds lower than 0.5
+    search_pdb_by_alignment: False
```

Comment:
I would change this documentation to the following:

Set to false to search the PDB Seqres database using jackhmmer from the target sequence only (more stringent). Set to true to search the PDB seqres database using an HMM built from
the output monomer alignment (less stringent). Warning: searching by HMM may result in crystal structures from very distant homologs or even unrelated sequences. The domain_threshold and sequence_threshold should be decreased to find distant homologs using HMMsearch.